static/app/views/metrics/metricListItemDetails.tsx

```diff
@@ -122,11 +124,23 @@ export function MetricListItemDetails({
         <MetricName>
           {/* Add zero width spaces at delimiter characters for nice word breaks */}
           {formatMRI(metric.mri).replaceAll(/([\.\/-_])/g, '\u200b$1')}
+          {isDuplicateWithDifferentUnit && (
+            <Disclaimer textColor="yellow400">
+              <IconWrapper>
+                <IconWarning color="yellow400" size="xs" />
+              </IconWrapper>
+              {t(
+                'Metrics with the same name and different units were detected. Disable any unwanted ones in settings.'
```

Comment:
The wording could be a bit clearer here I think: "Metrics with the same name and different units were detection. Unwanted metrics can be disabled in settings."

Reply:
sounds good!